deps/ed25519.ts

```diff
@@ -0,0 +1 @@
+export * from "https://esm.sh/@noble/ed25519@1.7.3"
```

Comment:
@tjjfvi Shall I update `examples/sign/ed25519.eg.ts` to use `ed25519` from `./deps`?

Reply:
Yes. Also it seems there's a newer version `2.0.0` –– will you please use that, assuming it doesn't break the usage / the new API is reasonable.